### ISSUE TITLE
Improve foam-template with easy way to get started with a note taking system

### ISSUE THREAD

**Issue by riccardoferretti** [foambubble/foam#447]
Currently it's pretty common for new users to feel a bit lost about how to use Foam once they have followed the steps in the foam template.

For someone new to note taking, having a reference point to get started with, can be the difference between continuing or dropping the effort.

There are many different systems one could use, and giving plenty of choices becomes a chore in its own right.
The goal of Foam is to be flexible, but provide sensible defaults.

I was thinking of including in the foam template some instructions to get started with [PARA](https://fortelabs.co/blog/para/), which is simple enough and good enough for most use cases.

This issue has two goals:
1. discuss whether PARA would be a good system to default to in Foam
2. track the work on the foam template to set up the PARA system. This could include:
  - create the various folders
  - add steps in the onboarding/getting started docs to use PARA
  - provide instructions on how to not use PARA but go with one's own approach

Again, **the goal is help users get started, not to impose on them a note taking system**. 
As their needs evolve (or if they are sophisticated enough to know what they want), they can use their own method.

**Comment by ingalless** [comment]
Chiming in some morning initial thoughts!

I love this. Will be really good to answer common questions right in the template. Perhaps it would be a good idea to try dig out the common questions from the discord chat?

Whatever we end up going with, I think it should be easy to "bail out" of, which I think you've eluded to. If a user works out in 10 minutes what he's doing then he shouldn't need to spend the next 20 deleting all our boilerplate!

Just some quick initial thoughts 👍

**Comment by SanketDG** [comment]
> Whatever we end up going with, I think it should be easy to "bail out" of, which I think you've eluded to. If a user works out in 10 minutes what he's doing then he shouldn't need to spend the next 20 deleting all our boilerplate!

I agree with @ingalless here, one should be able to build on "top of", not against.

I think keeping a minimal publishing system with recipes/plugins/templates for other systems would work well for us, but then again I have always like Foam's opinionated way of doing things.

About the template becoming easier to get started, I was wondering if there is any interest in making it more accessible, to the point where one should not even be aware of the SSG they are using underneath. Maybe a Dockerfile to start things off and later a bootstrap script?

**Comment by riccardoferretti** [comment]
@SanketDG for me there are two things that unfortunately atm are very tightly coupled together, but I am keen to disentangle:
1. Foam template as the initial collection of notes and settings that one can use to create a knowledge base
2. Publishing "recipes" that can be used to quickly publish a template to a specific target

The current template embeds a publishing recipe for GH Pages, so the two aspects are blended.
I think the right approach to decouple the two is still unclear.

In general my idea would be to have flexibility but with opinionated defaults.

All of the above being said, this issue is _only_ about point n. 1, the goal of PARA would be to get people to quickly write notes. The rest of the template would not be affected by this change, and part of the change would include the info to get rid of the PARA approach and start from scratch.

I think your comment is very valid and well in line with point n.2 above, how about we pull it into its own issue?

**Comment by FrancoB411** [comment]
Please no PARA. The fewer ideas, frameworks, and metaphors introduced, the easier for someone to onboard.

Users already have to learn VS CODE, Markdown, and wikilinks. That's a lot to learn compared to a pencil & notebook!

Understanding what users want to accomplish could be useful. Do they want to remember things? Do they want to write? Build Documentation?

The idea of looking through discord servers for pain points and common questions is a great place to start. But before creating an onboarding, it might make sense to talk to users and figure out what they hope to achieve, then create an onboarding that shows them how to get it.

Thoughts?
